src/stories/Modal.story.vue

```diff
@@ -1,17 +1,40 @@
 <template>
-  <Story title="Modal">
-    <HoppSmartModal :show="show" :title="'Modal Title'" @hide-modal="show = false" @resolve="resolveConfirmModal" />
+  <Story title="Modal (Deprecated)">
+    <button @click="showModal()">Show Modal</button>
+
+    <HoppSmartModal
+      :show="show"
```

Comment:
The button does not invoke the modal and the modal cannot be closed. To fix this, use a `v-if="show"` here and use the close prop to close the modal